CORTEX_M0+_RP2040/CMakeLists.txt

```diff
@@ -8,8 +8,6 @@ include(pico_sdk_import.cmake)
 
 PROJECT(examples)
 
-set( FREERTOS_KERNEL_PATH "../../../../../Source" )
```

Comment:
Each subproject configures the `FREERTOS_KERNEL_PATH`. Setting it here overrides the subproject logic and makes it impossible to use environment variables or command line options to set this value.